LondonBritishLibrary/orient/BLorient500.xml

```diff
@@ -69,17 +69,78 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     
                                 </support>
                                 <extent>
-                                    <measure unit="leaf" quantity="68">68</measure>
+                                    <measure unit="leaf" quantity="68">68+2</measure>
```

Comment:
should it then become 70? just asking

Reply:
the value of `@quantity`, you mean? Yes, that would make sense. I had actually never really paid attention to `@quantity` in `<measure>` (it's only used in these old BL records and some Ethio-SPaRe records, and not present in any example in the guidelines). Would it be useful to systematically add quantity in all records?
(This old BL foliation system and the numbering of the digital images is really unfortunate...)

Reply:
in E-S records the numbers usually add up ) this is why i asked

Reply:
But this is again the question what is counted and where; the E-S foliated starting from 1 from the first available page, so from flyleaves, whereas others usually do not number flyleaves, and describe them as part of binding. So am not really sure what the rule should be. Just numbers not adding up are confusing.

Reply:
I think quantity is good when it is there, one could then filter for mss with e g more than 100 leaves or less than 20 etc. things you cannot do if you do not have a clear numeric value for an attribute.

Reply:
I've added it here, but again, this has been used practically nowhere (only a few ES manuscripts and Zhenia's old BL descriptions). If it is so important, it should be included in the guidelines. Filtering according to leaf number seems not to be possible right now? But maybe I missed it.